cmd/podman/images/search.go

```diff
@@ -182,7 +182,11 @@ func imageSearch(cmd *cobra.Command, args []string) error {
 			listTagsEntries := buildListTagsJSON(searchReport)
 			return printArbitraryJSON(listTagsEntries)
 		}
-		rpt, err = rpt.Parse(report.OriginPodman, "{{range .}}{{.Name}}\t{{.Tag}}\n{{end -}}")
+		if cmd.Flags().Changed("format") {
+			rpt, err = rpt.Parse(report.OriginUser, searchOptions.Format)
```

Comment:
Why isn't this OriginPodman?

Reply:
Because a user gave us the format, OriginPodman is only for built in templates that podman uses. A user template does not include the header for example.

Reply:
According to the description at https://github.com/containers/common/blob/412dafe2b5454b344a3ac2aea840cb2a9951c238/pkg/report/formatter.go#L66, when the user provides the "format" parameter, it should be used as "OriginUser".

Reply:
SGTM